src/ListBox/listBoxOption.css

```diff
@@ -1,42 +1,103 @@
-@import '../proto/definitions/_colors.css';
+@import '../proto/definitions/variables.css';
 
 .default
 {
-    display:    flex;
-    margin:     0;
-    padding:    1rem;
+    display:        flex;
+    align-items:    center;
+    margin:         0;
 
-    margin:     0;
-    padding:    1rem;
+    padding-left:           var( --spacing-2 );
+    padding-right:          var( --spacing-2 );
+    padding-top:            var( --spacing-1 );
+    padding-bottom:         var( --spacing-1 );
 
     cursor:     pointer;
 
+    &:hover,
+    &:hover .optionText,
+    &:hover .description
+    {
+        background-color:   var( --PC-BLUE );
+        color:              var( --PC-WHITE );
+    }
+
```

Comment:
I think these selectors are too vague. It would be better to be a bit more specific here:
```
&:hover
{
    background-color:   var( --PC-BLUE );

    .optionText,
    .description
    {
        color: var( --PC-WHITE );
    }
}
```

Also `className = { cssMap.description }` is missing from the description Text component in `ListBoxOption.jsx`.